### DISCUSSION TITLE
It is always waiting for the screen

### DISCUSSION THREAD

**Discussion by ikunpy** [rustdesk/rustdesk#5182] [General]
### Bug Description

This seems strange. Previously, I used RustDesk on the campus network a thousand kilometers away from here, but when I brought the charged computer home, I found that I couldn't connect to this computer properly. It always displayed "Successfully connected, waiting for picture return", but always waiting for picture return.

I am very confused about this.

At the same time, I found a very interesting thing. When I modify the network connection, RustDesk can connect to the controlled computer in a short period of time, but after a few seconds to minutes, it will automatically disconnect, and when reconnected, it will always wait for the screen to return, which looks very strange.

Meanwhile, I speculate that there may be a problem with the DNS resolution service, but when I locked the DNS server to services provided by Tencent and Baidu, the problem was not resolved
China.

### How to Reproduce

Connect to the campus network of colleges and universities in Chinese Mainland, and then try another residential network。

### Expected Behavior

Solve the problem

### Operating system(s) on local side and remote side

Win 10 professional

### RustDesk Version(s) on local side and remote side

1.1.9

### Screenshots

![1690552415776](https://github.com/rustdesk/rustdesk/assets/113223580/74e1ade5-ecec-4f32-a2e9-2032ff168cd3)

**Comment by rustdesk** [comment]
very sorry, we do not have servers in China, bandwidth too expensive. no way to help you solve your network issue.

**Comment by dinger1986** [comment]
why not deploy your own server @ikunpy
